docs/2.guide/2.directory-structure/1.server.md

```diff
@@ -14,7 +14,7 @@ Nuxt automatically scans files inside these directories to register API and serv
 
 Each file should export a default function defined with `defineEventHandler()` or `eventHandler()` (alias).
 
-The handler can directly return JSON data, a `Promise` or use `event.node.res.end()` to send a response.
+The handler can directly return JSON data, a `Promise`, a `Response`, or use `event.node.res.end()` to send a response.
```

Comment:
might be worth instead mentioning `h3`'s `send` util?

Reply:
send does not handles response (altohugh we might suggest `event.respondWith` with upcoming h3, i prefer we use direct responses. there is also typing concerns that we lose with response by advocating it...)

Reply:
I was suggesting dropping `event.node.res.end()`

Reply:
Makes sense. (docs changes can be outside of this pr considering h3 is first to be released and available for all users before nuxt 3.7 we can update docs directly 👍🏼